app/views/pages/index.html.erb

```diff
@@ -4,14 +4,17 @@
   <div class="img-fluid m-0 py-5 px-2 rounded"
         style="background: url(<%= asset_path('_78A0587-nb.jpg') %>) no-repeat center; background-size: cover;">
     <div class="container-fluid mt-4">
-      <% Site.joins(:rooms).where('display_on_home_page = ?', true).distinct.order(:position).each do |site| %>
+      <% Site.joins(:rooms).where('display_on_home_page = ? and status != ?', true, Room.statuses[:passive]).distinct.order(:position).each do |site| %>
```

Comment:
Maybe we could create a scope to do this for us

Reply:
https://api.rubyonrails.org/classes/ActiveRecord/SpawnMethods.html#method-i-merge may help you

Reply:
I let it that way because I really think, this needs an update, and move `display_on_home_page` on site.

But I can do it anyway

Reply:
As you want